README.md

```diff
@@ -141,17 +136,48 @@ experimentation while writing code that is production-ready from the get-go.
 - 🛫 ZenML enables MLOps infrastructure experts to define, deploy, and manage
 sophisticated production environments that are easy to use for colleagues.
 
-![The long journey from experimentation to production.](/docs/book/.gitbook/assets/intro-zenml-overview.png)
-
-ZenML provides a user-friendly syntax designed for ML workflows, compatible with
-any cloud or tool. It enables centralized pipeline management, enabling
-developers to write code once and effortlessly deploy it to various
-infrastructures.
-
 <div align="center">
-    <img src="docs/book/.gitbook/assets/overview.gif">
+  <img width="60%" src="/docs/book/.gitbook/assets/zenml-hero.png" alt="ZenML Hero"/>
 </div>
 
+ZenML aims to optimize your entire machine learning lifecycle - you'll be 80% faster to production, generate 3x more models, and massively reduce your cloud spend.
```

Comment:
maybe instead of "aims to", something more active? 

- ZenML optimizes your entire...
- ZenML is all about optimizing your...
- etc

Reply:
also feels like maybe these very precise numbers need links to user case studies etc. without links it feels a bit arbitrary? (Not sure how others would read it, though).